app/main.py

```diff
@@ -1,6 +1,37 @@
+import os
+from datetime import datetime as dt
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+BASE_URL = "https://api.weatherapi.com/v1/current.json"
+API_KEY_PARAM = "key"
+CITY_PARAM = "q"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+    if not api_key:
+        raise ValueError("No API key found in environment variables")
+
+    url = f"{BASE_URL}?{API_KEY_PARAM}={api_key}&{CITY_PARAM}={CITY}"
```

Comment:
The way you construct the URL is correct, but you could improve the readability of your code by using the `params` argument of the `requests.get()` function. This argument takes a dictionary of parameters to append to the URL. Here is an example of how you could do it: `requests.get(BASE_URL, params={API_KEY_PARAM: api_key, CITY_PARAM: CITY})`. This way, you don't have to manually construct the URL and it's easier to add or remove parameters.